Store/DataStoreModule.py

```diff
@@ -534,3 +579,32 @@ def getTabletypeData(self, tabletypes):
             for table in self.metaClasses[tabletype]:
                 retmap[table.__name__] = self.session.query(table).count()
         return retmap
+
+    # Populate methods in order to import CSV files
+
+    def populateReference(self, reference_data_folder=None):
+        """Import given CSV file to the given reference table"""
+        if reference_data_folder is None:
+            reference_data_folder = os.path.join("..", "default_data")
+
+        files = os.listdir(reference_data_folder)
+        reference_files = [file for file in files if file[:-4] in REFERENCE_TABLES]
```

Comment:
we should have a comment to explain :-4 here please.  If it's to remove the file suffix, I'm afraid we need a more sophisticated version, to allow for suffixes that are longer or shorter than 3 chars.

Reply:
I added TODO for that. I think we need remove_suffix method. I'll do some search before implementing it

Reply:
It was easier than I thought. There is alread built-in function for removing extensions. I added it.

https://github.com/debrief/python-import-library/pull/15/commits/c502bdc093880e6c3738d65a0ef8123a9ed3f333